.github/workflows/ci.yml

```
@@ -376,4 +394,25 @@ jobs:
         run: composer bin psalm install
 
       - name: Run static analysis
-        run: bin/tools/psalm/vendor/vimeo/psalm/psalm --output-format=github --php-version=7.4
+        run: bin/tools/psalm/vendor/vimeo/psalm/psalm --output-format=github --php-version=7.2
+
+  docker-stack:
+    name: CI with docker stack
+    continue-on-error: true
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          # - windows-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: docker-practice/actions-setup-docker@master
+        if: ${{ matrix.os == 'macos-latest' }}
+
+      - name: Checkout code
+        uses: actions/checkout@v2.3.3
+
+      - name: Run test suite with docker
+        run: |
+          make tests
```

```suggestion
          make validate
```

Check the gh actions, this fails but doesn't fail the job..?

---

yep I added `continue-on-error: true` to see the full result of the matrix. I've just removed it